e2e-tests/test-applications/nextjs-test-app/src/app/globals.css

```diff
@@ -0,0 +1,42 @@
+:root {
```

Comment:
I would probably just remove the CSS files. They shouldn't add anything to the tests.